AWS-CodePipeline/sam-monorepo-pipeline-template/questions.json

```diff
@@ -0,0 +1,214 @@
+{
+    "questions": [
+        {
+            "question": "What is subfolder path of your project from the root of the repository?",
+            "key": "monorepo_project_subfolder",
+            "isRequired": true
+        },
+        {
+            "question": "What is the template file path?",
+            "key": "sam_template",
+            "default": "template.yaml"
+        },
+        {
+            "question": "We use the stage configuration name to automatically retrieve the bootstrapped resources created when you ran `sam pipeline bootstrap`.\n",
+            "key": "message_test_stage_name",
+            "kind": "info"
+        },
+        {
+            "key": "message_list_stage_names_testing",
+            "question": {
+                "keyPath": [
+                    "stage_names_message"
+                ]
+            },
+            "kind": "info"
+        },
+        {
+            "key": "test_stage_name",
+            "question": "Select an index or enter the stage 1's configuration name (as provided during the bootstrapping)",
+            "allowAutofill": true,
+            "isRequired": true
+        },
+        {
+            "key": "test_stack_name",
```

Comment:
Other templates use `testing_` as prefix to keys of the testing stage. Making it consistent with them avoid confusing to future contributors.

Reply:
Good point. I changed from `testing_` to `test_` simply so that both stages (`test` and `prod`) had the same number of characters.
I'll revert it back to `testing_` for consistency. Thanks for pointing it out.

Reply:
Fixed.